.github/workflows/release.yml

```diff
@@ -172,8 +176,10 @@ jobs:
         helm repo index --merge index.yaml build-archives/helm
         mv build-archives/helm/index.yaml ./index.yaml
         cp -r build-archives/helm/. ./
+        # update version in install script
+        sed -i 's/LINKERD_SMI_VERSION:-.*/LINKERD_SMI_VERSION:-"${{needs.gh_release.outputs.tag}}"}/' ./install
         git config user.name github-actions
         git config user.email github-actions@github.com
-        git add linkerd-smi-*.tgz index.yaml
+        git add linkerd-smi-*.tgz index.yaml install
         git commit -sm "Add new Helm Chart"
```

Comment:
Now that we have access to the tag, can you make this commit message more explicit by including that tag, like `"Add new Helm Chart vx.xx"`?

Reply:
yep, good idea!